rotkehlchen/tests/unit/decoders/test_yearn.py

```diff
@@ -191,8 +193,227 @@ def test_withdraw_yearn_v3(
     ]
 
 
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x34A7a276eD77c6FE866c75Bbc8d79127c4E14a09']])
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_deposit_yearn_v2(
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x7e6493bb2e0c09b9986ab43a562409fd2cf99caca3d1b65fa4b0d46ef5588081')  # noqa: E501
+    get_or_create_evm_token(
```

Comment:
why do you need this? Does the decoder not do get_or_create? Samefor other tests.

Reply:
Normally the vault tokens are all created in `query_yearn_vaults` using the data from the API, but in the tests it doesn't do that api call, so we have to manually add the token needed for the test, unless its a token that is already in the packaged globaldb, as is the case for some of the older tests.